client/src/components/util/truncate.tsx

```diff
@@ -89,9 +89,22 @@ export default (props: any) => {
       "aria-label": originalString,
     })
   );
+  // we need an ID to check for this content, since this is the only place the geneset description appears
+  const descriptionContent = (
+    <span
+      test-id={`geneset-description-tooltip-${originalString}${tooltipAddendum}`}
```

Comment:
why not just put the test-id on the tooltip element (does it not propagate attributes?)

Reply:
My tests of that showed they are not propegated